src/decompression.jl

```diff
@@ -256,9 +256,9 @@ function in_triangle(i::Integer, j::Integer, uplo::Symbol)
     if uplo == :F
         return true
     elseif uplo == :U
-        return i <= j
-    else
-        return i >= j
+        return i ≤ j
```

Comment:
```suggestion
        return i <= j
```
we didn't use unicode anywher else, so I'd prefer us to remain consistent